src/Appwrite/Platform/Workers/Mails.php

```diff
@@ -58,13 +58,21 @@ public function action(Message $message, Registry $register): void
         $subject = $payload['subject'];
         $variables = $payload['variables'];
         $name = $payload['name'];
-        $body = Template::fromFile(__DIR__ . '/../../../../app/config/locale/templates/email-base.tpl');
+        $body = $payload['body'];
 
+        $bodyTemplate = Template::fromFile(__DIR__ . '/../../../../app/config/locale/templates/email-base.tpl');
+        $bodyTemplate->setParam('{{body}}', $body);
         foreach ($variables as $key => $value) {
-            $body->setParam('{{' . $key . '}}', $value);
+            $bodyTemplate->setParam('{{' . $key . '}}', $value);
         }
+        $body = $bodyTemplate->render();
 
-        $body = $body->render();
+        $subjectTemplate = Template::fromString($subject);
```

Comment:
not sure where `$subject` is coming from in case of non certificate emails. $subject is no longer sent in the payload